lib/PlacesProvider.js

```diff
@@ -187,14 +187,15 @@ Links.prototype = {
    *
    * @param {Object} options
    *        {Integer} limit: Maximum number of results to return. Max 20
-   *        {Milliseconds} limitDate: Only return links fresher than limitDate
+   *        {Milliseconds} beforeDate: Only return links older than beforeDate
+   *        {Milliseconds} afterDate: Only return links fresher than afterDate
```

Comment:
afterDate is used by PlacesProvider.getFrecentLinks() - recall we wanted to get last 72 hours worth of links ordered by frecency